restli-server-extras/src/main/java/com/linkedin/restli/server/ParseqTraceDebugRequestHandler.java

```diff
@@ -297,7 +297,7 @@ private static String createTraceRenderScript(String trace)
     String result = String.format(
         TRACE_RENDER_SCRIPT,
         TRACE_RENDER_FUNCTION,
-        StringEscapeUtils.escapeJavaScript(trace));
+        StringEscapeUtils.escapeEcmaScript(trace));
```

Comment:
Looking at this [StackOverflow](https://stackoverflow.com/a/29146537) research, it looks like indeed `escapeEcmaScript` is the appropriate replacement for `escapeJavaScript`, so this change should be compatible 🙏